k8s/arroyo/templates/configmap.yaml

```diff
@@ -59,6 +59,10 @@ data:
           {{- end }}    
         image: "{{ .Values.worker.image.repository }}:{{ .Values.worker.image.tag }}"
         image-pull-policy: "{{ .Values.worker.image.pullPolicy }}"
+        image-pull-secrets:
+        {{- with .Values.imagePullSecrets }}
```

Comment:
`image_pull_secrets` is defined as a string, but here it's passed in an object which fails:
```
Configuration is invalid!
  • invalid type: found unit, expected a string for key "default.kubernetes-scheduler.worker.image-pull-secrets" in /root/.config/arroyo/config.yaml YAML file
```